OpenRA.Mods.Common/Traits/LinkClientBase.cs

```diff
@@ -0,0 +1,54 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	public abstract class LinkClientBaseInfo : ConditionalTraitInfo, ILinkClientInfo, Requires<ILinkClientManagerInfo> { }
+
+	public abstract class LinkClientBase<InfoType> : ConditionalTrait<InfoType>, ILinkClient, INotifyCreated where InfoType : LinkClientBaseInfo
+	{
+		readonly Actor self;
+
+		public abstract BitSet<LinkType> GetLinkType { get; }
```

Comment:
```suggestion
		public abstract BitSet<LinkType> LinkType { get; }
```

(This is not a method.)

Reply:
Then the name matches the class name, though I suppose it's not a style error